### ISSUE TITLE
brew outdated Error: Unexpected method 'appcast' called on Cask cryptobridge

### ISSUE THREAD

**Issue by faxotherapy** [Homebrew/homebrew-cask#173444]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [X] I have retried my command with `--force`.
- [X] I ran `brew update-reset && brew update` and retried my command.
- [X] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match / SHA256 mismatch](https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match) error (do not open an issue before trying to open a PR to fix first).

### Description of issue

Running `brew outdated` outputs the following:
```
Error: Unexpected method 'appcast' called on Cask cryptobridge.
```

### Command that failed

brew outdated

### Output of command with `--verbose --debug`

```shell
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/rubberband
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jnettop
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/midnight-commander
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cjson
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/bat-extras
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sponge
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/faad2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/autossh
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/z3
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/imlib2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/hwloc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lame
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pkg-config
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/vim
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tree
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/regex-opt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libproxy
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libheif
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jdupes
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libtiff
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libtool
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/coreutils
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pv
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/yasm
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jansson
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gmp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/aribb24
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dust
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/minisign
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lzlib
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/wget
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sd
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libebml
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ffmpeg
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/unpaper
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fdk-aac
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pinentry
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/xorgproto
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/json-c
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/two-lame
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libplacebo
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fdupes
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lzip
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libidn2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libunibreak
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/parallel
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/magic-wormhole
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libmagic
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/wxwidgets
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pigz
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxml2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zbar
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pandoc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/vapoursynth
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/xvid
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python-packaging
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/shaderc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libksba
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lolcat
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ilmbase
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/autojump
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/exiftool
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nghttp2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gpgme
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxau
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/little-cms2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/llvm@17
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lynx
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/go
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lbzip2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zsh-autosuggestions
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dmg2img
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxtst
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/shc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libpng
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ncurses
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ghostscript
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pygments
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python-setuptools
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/frei0r
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gtk+3
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jsoncpp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sub2srt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nettle
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pixman
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gdbm
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mpdecimal
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pycparser
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxdmcp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/progress
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/netcat
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gnu-getopt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cmake
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/qt@5
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/opus
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libuv
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/popt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/freetype
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/docutils
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mpfr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/assimp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/yt-dlp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/rtmpdump
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/imath
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cmocka
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/trash-cli
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gnu-tar
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pipx
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/liblinear
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/terrapin-scanner
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/p7zip
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/md4c
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pidof
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/highlight
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libsamplerate
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libvorbis
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cryptography
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/espeak
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/boost
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pybind11
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libavif
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gts
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mpg123
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/smartmontools
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fmt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libunistring
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/liblqr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libmediainfo
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/grc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pango
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/texi2html
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openjpeg
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/highway
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libvo-aacenc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libyaml
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/xmlto
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ldns
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/wifi-password
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zimg
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jemalloc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/brotli
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/speedread
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gdk-pixbuf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cmatrix
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/bdw-gc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/speex
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libogg
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libslirp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/icu4c
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/neofetch
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fd
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/shared-mime-info
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mtr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zeromq
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cffi
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dnstracer
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/perl
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/timedog
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/harfbuzz
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sleepwatcher
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python@3.12
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openssh
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libbluray
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/glib
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/rust
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxcb
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zsh-completions
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ripgrep-all
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/readline
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/exa
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python-psutil
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libcerf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nlohmann-json
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/netpbm
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/protobuf-c
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dnstop
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ipcalc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/at-spi2-core
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ffmpeg@6
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openvino
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/hunspell
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/llvm
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/aom
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/qt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/img2pdf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mbedtls
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libde265
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lz4
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/s-lang
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ocrad
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/yara
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tbb
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/rsync
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libraqm
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tesseract-lang
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nasm
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/geoip
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/hstr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/graphite2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dbus
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ssh-audit
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nss
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/wireguard-go
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gzip
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libcaca
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libarchive
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gcc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libassuan
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/srt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/berkeley-db@5
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mujs
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zopfli
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/screenresolution
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/recode
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tesseract
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cairo
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/webp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sqlite
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libgpg-error
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/c-ares
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gobject-introspection
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/protobuf@21
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/iperf3
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ttfautohint
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fribidi
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python-tabulate
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sshfs
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/svt-av1
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libpthread-stubs
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/xz
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nethogs
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/numpy
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libnghttp2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/vde
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromTapLoader): loading homebrew/dupes/grep
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading grep
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lua
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libusb-compat
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/bash
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ninja
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/utf8cpp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python-lxml
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openexr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dateutils
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libgcrypt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gocr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fontconfig
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/double-conversion
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/htop
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/procs
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openjdk
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/toilet
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mad
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/meson
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxfixes
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openblas
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/giflib
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gifski
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/docbook-xsl
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/media-info
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pillow
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/certifi
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/capstone
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ghc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libmatroska
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jbig2enc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libcroco
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/iftop
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/speedtest-cli
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/rig
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libfido2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pbzip2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dtc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gnu-sed
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mkvtoolnix
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libvpx
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxrender
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/brew-cask-completion
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/librist
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lftp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libb2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/snappy
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/flac
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gpac
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tor
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/figlet
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gifsicle
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pngquant
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/links
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libvmaf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openssl@3
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gawk
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ca-certificates
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gettext
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/luajit-openresty
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libssh2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libmicrohttpd
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dcraw
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pdftilecut
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/keychain
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/librsvg
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libcbor
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/qemu
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zstd
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tcl-tk
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/alass
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tasksh
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sphinx-doc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libsodium
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jbig2dec
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/dav1d
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mpv
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/task
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/autoconf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/plzip
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/csvtomd
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libscrypt
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/shellcheck
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/opencore-amr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tmux
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/rav1e
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxext
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mcfly
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/hicolor-icon-theme
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/mplayer
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pth
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/m4
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/glab
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libtasn1
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/streamlink
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/watch
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/oniguruma
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libraw
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gd
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/isl
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/qrencode
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/vifm
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/xxhash
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python-typing-extensions
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libimagequant
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libssh
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/vulkan-loader
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ffmpeg@4
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pcre2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libevent
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/rename
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/btop
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openssl@1.1
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/bat
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nnn
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/leptonica
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pcre
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cloc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/x264
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libepoxy
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libgit2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jpeg-xl
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python@3.10
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/luajit
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/aescrypt-packetizer
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libidn
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pugixml
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/corrosion
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/grsync
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libvidstab
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/docbook
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/cabal-install
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/doxygen
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ocrmypdf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/faac
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/portaudio
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/vnstat
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/glow
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/six
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sshs
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/x265
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/reattach-to-user-namespace
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/djvulibre
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/node
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pdf2image
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libsoxr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libomp
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/unbound
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/diff-pdf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libzen
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gifify
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/opusfile
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jpeg
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ncdu
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tldr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nmap
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ssh-copy-id
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libmms
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libzip
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/arp-scan
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zsh
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/theora
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jq
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/wireguard-tools
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/zsh-syntax-highlighting
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/uchardet
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/poppler
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/openldap
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sdl2
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/spoof-mac
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libxi
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/vulkan-headers
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromTapLoader): loading cjbassi/gotop/gotop
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/nspr
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromTapLoader): loading homebrew/core/python-mutagen
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromKegLoader): loading python-mutagen
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libusb
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python@3.11
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/qpdf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ripgrep
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libmng
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libass
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python@3.8
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/a52dec
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fsevent_watch
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tcpdump
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ansifilter
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/pdfgrep
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/imagemagick
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/guetzli
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/findutils
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/guile
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jpegoptim
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gnupg
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libsndfile
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jasper
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/p11-kit
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libev
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libx11
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/protobuf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/unar
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fswatch
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/ruby
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/excel-compare
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/fzf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromTapLoader): loading homebrew/core/python-brotli
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromKegLoader): loading python-brotli
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/automake
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/abseil
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/git
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/lzo
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/glances
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/djvu2pdf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libffi
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/telnet
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/python@3.9
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/jpeg-turbo
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sdl12-compat
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libmpc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/duf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gnutls
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/gsettings-desktop-schemas
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/berkeley-db
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromTapLoader): loading homebrew/core/python-markupsafe
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromKegLoader): loading python-markupsafe
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromTapLoader): loading yudai/gotty/gotty
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/libpcap
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/briss
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/npth
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/adns
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/duktape
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading alacritty
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading atom
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading betterzip
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading bitbar
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading buzz
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading cakebrew
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading coccinellida
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromInstalledPathLoader): loading cryptobridge
Error: Unexpected method 'appcast' called on Cask cryptobridge.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading fluor
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromNameLoader): loading font-fantasque-sans-mono
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromNameLoader): loading font-victor-mono
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading freetube
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromInstalledPathLoader): loading icecat
Error: Unexpected method 'appcast' called on Cask icecat.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading iina
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading kap
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading librewolf
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading mark-text
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading qlcolorcode
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading qlimagesize
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading qlmarkdown
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading qlstephen
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading qlvideo
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading quicklook-csv
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading rclone-browser
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading secure-pipes
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading stats
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading subsync
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading vidcutter
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading vienna
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading vivaldi
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading vlc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading webpquicklook
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromTapLoader): loading homebrew/dupes/grep
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading grep
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromTapLoader): loading homebrew/dupes/grep
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading grep
grep (2.27, 2.28) < 3.11 [pinned at 2.28]
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/sshfs
sshfs (2.10_2) < 3.7.3 [pinned at 2.10_2]
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromAPILoader): loading homebrew/core/tmux
tmux (3.0a) < 3.4_1 [pinned at 3.0a]
```
```


### Output of `brew doctor` and `brew config`

```shell
brew doctor:

Error: Unexpected method 'appcast' called on Cask cryptobridge.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Error: Unexpected method 'appcast' called on Cask icecat.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Error: Unexpected method 'appcast' called on Cask cryptobridge.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Error: Unexpected method 'appcast' called on Cask icecat.
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed casks are deprecated or disabled.
You should find replacements for the following casks:
  atom
  buzz
  cakebrew
  rclone-browser

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed manually.
You should find replacements for the following formulae:
  grep
  python-mutagen
  python-brotli
  python-markupsafe

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  exa
  geoip
  ilmbase
  neofetch
  openssl@1.1
  python-psutil
  python-typing-extensions
  python@3.8

Warning: You have the following deprecated, official taps tapped:
  Homebrew/homebrew-cask-versions
Untap them with `brew untap`.

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  glib
  pdf2image
```

brew config:
```
HOMEBREW_VERSION: 4.2.21-88-gf590dad
ORIGIN: https://github.com/Homebrew/brew
HEAD: f590dadbf22b451a42896364ef0554503409abe0
Last commit: 16 minutes ago
Core tap HEAD: 93767ae41b39072dc55ea1d1ce2424344e4231ef
Core tap last commit: 5 minutes ago
Core tap JSON: 10 May 14:01 UTC
Core cask tap HEAD: a4e5b72ee974ae75aabd5c1297c12cf271112ca9
Core cask tap last commit: 66 minutes ago
Core cask tap JSON: 10 May 14:01 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: ["--appdir=~/Applications/"]
HOMEBREW_EDITOR: /usr/local/bin/vim
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ANALYTICS: set
HOMEBREW_NO_AUTO_UPDATE: set
HOMEBREW_NO_INSTALL_CLEANUP: set
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit haswell
Clang: 14.0.0 build 1400
Git: 2.45.0 => /usr/local/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 12.7.1-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
```
```


### Output of `brew tap`

```shell
cjbassi/gotop
homebrew/bundle
homebrew/cask
homebrew/cask-fonts
homebrew/cask-versions
homebrew/core
homebrew/services
yudai/gotty
```

**Comment by bevanjkay** [comment]
Duplicate of https://github.com/Homebrew/homebrew-cask/issues/173201
